### ISSUE TITLE
[Bug]: Proxying videos through Invidious with HTTP Auth doesn't work

### ISSUE THREAD

**Issue by GamePlayer-8** [FreeTubeApp/FreeTube#4474]
### Guidelines

- [X] I have encountered this bug in the [official downloads of FreeTube](https://github.com/FreeTubeApp/FreeTube#official-downloads).
- [X] I have encountered this bug in the [latest release of FreeTube](https://github.com/FreeTubeApp/FreeTube/releases).
- [X] I have searched the issue tracker for [open](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aopen+is%3Aissue) and [closed](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aissue+is%3Aclosed) issues that are similar to the bug report I want to file, without success.
- [X] I have searched the [documentation](https://docs.freetubeapp.io/) for information that matches the description of the bug I want to file, without success.
- [X] This issue contains only one bug.

### Describe the bug

1. Go to settings -> General Settings -> Current Invidious Instance.
2. Set an instance what's under a HTTP Auth (use scheme like so `https://<user>:<password>@<domain>`).
3. Go to settings -> Player Settings -> Enable `Proxy Videos Through Invidious.
4. Play a random video.

The error will appear:
```shell
TypeError: 
Failed to execute 'fetch' on 'Window': 
Request cannot be constructed from a URL that includes credentials: 
https://*user*:*password*@*domain*/api/v1/videos/*VIDEO_ID*
```

### Expected Behavior

Video should load and be playable.

### Issue Labels

accessibility issue

### FreeTube Version

v.0.19.1 Beta

### Operating System Version

Artix Linux Rolling on KDE Plasma

### Installation Method

AUR (Unofficial)

### Primary API used

Invidious API

### Last Known Working FreeTube Version (If Any)

-

### Nightly Build

- [ ] I have encountered this bug in the latest [nightly build](https://docs.freetubeapp.io/development/nightly-builds).

**Comment by absidue** [comment]
FreeTube doesn't support authentication for Invidious, so those errors are entirely expected, which means this is a feature request not a bug report. Please close this issue and open a new one with a feature request.

**Comment by GamePlayer-8** [comment]
sure.

**Comment by GamePlayer-8** [comment]
Feature request should be available under https://github.com/FreeTubeApp/FreeTube/issues/4475

**Comment by EchedelleLR** [comment]
> FreeTube doesn't support authentication for Invidious, so those errors are entirely expected, which means this is a feature request not a bug report. Please close this issue and open a new one with a feature request.

:/
